hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -154,6 +127,18 @@ read_env constraints.sdc.env
 touch $USER_ENV_FILE
 read_env $USER_ENV_FILE
 
+# RI's recommended approach to filter out unnecessary errors on async_fifo
+set async_fifo [get_cells -hsc . -hier * -filter {ref_name == prim_fifo_async}]
+catch {rm tmp.env}
+set outfile [open tmp.env  w]
+puts $outfile "set_data_clock_domain rdata_o -module prim_fifo_async -derived_from rvalid_o"
+puts $outfile "set_stable {{top_earlgrey.u_pinmux_aon.mio_oe_retreg_q[46:0]}}"
```

Comment:
Should this script reside in `hw/top_earlgrey` area, given that it references paths that may not be valid in other OT chips/tops?

Reply:
you are absolutely right, if you scroll up, there is an old TODO that says this should not be in this file and be moved elsewhere... the main reason why it's here is because the tools treat "constraints" and "waivers" differently. We can't place them where the waiver files are since then it has no effect.  So we need to add a separate variable and eventually split this up.

Reply:
Aah, ok.

Reply:
@jeoongp it will be good to come back to this later as a clean up item. I don't think it's super urgent, but @sriyerg is right that we should separate top specific stuff from the general setup (sorry I was the one who was originally guilty of this)

Reply:
@sriyerg @tjaychen thanks for good suggestion and sorry for late reply. I was looking for a simpler local path instead of a full path. You are right. I will separate top specific stuff or replace the full path with a relative local path.